<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>senales cerebrales</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="campo_grafico" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>561</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cargar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>cargar señal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Señal Cargada:</string>
    </property>
   </widget>
   <widget class="QWidget" name="campo_grafico2" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>440</y>
      <width>561</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>410</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Señal Filtrada:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="cargar_canal2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>730</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>427</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingresar canal a filtrar:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="canal_filtro">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>157</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="nivel_filtrado">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccione una opción...</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>One</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Single Level Noise </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Multiple Level Noise</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione el nivel de Denoising:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tipo_umbral">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>660</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sleccione una opción...</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Universal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Minimax</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sure</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>660</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione el umbral:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="forma_filtrado">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>590</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccione una opción...</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Duro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Suave</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>590</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccione la forma de Denoising:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guarda_filtrada">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>730</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>guardar_icon.png</normaloff>guardar_icon.png</iconset>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>376</width>
      <height>200</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Desplazamiento en el tiempo:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="boton_atras">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="boton_adelante">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Escalamiento en amplitud:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="boton_disminuir">
           <property name="text">
            <string>Disminuir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="boton_aumentar">
           <property name="text">
            <string>Aumentar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Ingrese el rango de tiempo:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Min:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tiempo_min"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Max:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tiempo_max"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="cargar_tiempo">
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>290</y>
      <width>421</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Ingresar canal a visualizar:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="canal_text">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cargar_canal">
       <property name="text">
        <string>Cargar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
